core/src/main/java/org/renjin/primitives/Jvmi.java

```diff
@@ -61,7 +61,7 @@ public static SEXP importClass(@Current Context context, @Current Environment rh
     }
 
     ExternalPtr ptr = new ExternalPtr(clazz);
-    rho.setVariable(Symbol.get(clazz.getSimpleName()), ptr);
+    rho.setVariable(context, Symbol.get(clazz.getSimpleName()), ptr);
```

Comment:
What is the correct behavior here?

```.R
f <- function(val) 42
makeActiveBinding("HashMap", f, environment())
import(java.util.HashMap)
assertThat(typeof(HashMap), identicalTo("double"))  
```
Or should the `import` function through an error if an active binding is defined?